scripts/globals/spells/raise.lua

```diff
@@ -16,6 +16,9 @@ function onSpellCast(caster,target,spell)
         if (target:getName() == "Prishe") then
             -- CoP 8-4 Prishe
             target:setLocalVar("Raise", 1)
+            target:entityAnimationPacket("sp00")
```

Comment:
@teschnei wasn't that animation already being handled elsewhere when you implemented these originally?

Reply:
er, on prishe and aern's I mean , not the spell

Reply:
One of the few things I changed was moving this stuff from Prishe to the spell because she wasn't accepting the raise before. It was in onMobFight which wasn't getting called while she was dead.

Reply:
onMobFight seems like a terrible place to have it